parachain/pallets/basic-channel/src/inbound/mod.rs

```diff
@@ -66,7 +66,7 @@ pub mod pallet {
 	pub type SourceChannel<T: Config> = StorageValue<_, H160, ValueQuery>;
 
 	#[pallet::storage]
-	pub type Nonce<T: Config> = StorageValue<_, u64, ValueQuery>;
+	pub type Nonces<T: Config> = StorageMap<_, Twox64Concat, H160, u64, ValueQuery>;
```

Comment:
I know this is subjective, but I prefer the singular form. If the data type were an actual or array/vector then nonces would make more sense. 

```suggestion
	pub type Nonce<T: Config> = StorageMap<_, Twox64Concat, H160, u64, ValueQuery>;
```

Reply:
I'm happy to rename it - should we also rename the mappings in the Parachain -> Ethereum direction?

Reply:
Made this change in #704 for consistency